CMakeLists.txt

```diff
@@ -19,11 +19,7 @@ if (ZeroMQ_FOUND AND (NOT TARGET libzmq OR NOT TARGET libzmq-static))
   message(FATAL_ERROR "ZeroMQ version not supported!")
 endif()
 
-if (EXISTS "${CMAKE_SOURCE_DIR}/.git")
```

Comment:
Note that this check is used by all other ZMQ projects which implement a draft system

Reply:
I see, in this case we can keep it consistent with others.